.kokoro/run/common.cfg

```diff
@@ -0,0 +1,22 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Download trampoline resources. These will be in ${KOKORO_GFILE_DIR}
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
```

Comment:
@fhinkel do you know if we need these other properties (besides the custom `env-vars` at the bottom?)

Reply:
We use them in the other config files, I think they're needed. If not, at least we're consistent and can remove them all at once.